backends/gdx-backends-gwt/res/com/badlogic/gwtref/GwtReflect.gwt.xml

```diff
@@ -49,7 +49,7 @@
 		value="com.badlogic.gdx.Net" />
 
 	<extend-configuration-property name="gdx.reflect.include"
-		value="com.badlogic.gdx.maps.MapObject" />
+		value="com.badlogic.gdx.maps" />
```

Comment:
Does this include sub-packages (and thus we could remove lines 53 to 56)?

Reply:
I'm pretty sure it does, since a line much earlier adds `com.badlogic.gdx.scenes.scene2d` to the cache, but that package has nothing in it directly, only sub-packages. I'll commit the change after I make sure it still runs the tests.